packages/grafana-data/src/field/fieldState.ts

```diff
@@ -143,6 +143,9 @@ export function calculateFieldDisplayName(field: Field, frame?: DataFrame, allFr
     displayName = getUniqueFieldName(field, frame);
   }
 
+  if (isComparisonSeries) {
```

Comment:
😬 Can we make it the responsibility of whoever produces this to set a name with "(comparison)" in it.  The magic name formats get really hard to decouple.

Reply:
@ryantxu not really, the naming happens after field overrides (which is very separate from query processing in scenes) so all that query/data processing can do is mark it as "this is a timeshifted comparison series", in order for the series to have the same name as the unshifted series.